### PR TITLE
Project clean-up and changelog expansion

### PR DESCRIPTION
**Pull request by minchinweb**
Some cleanup, but mostly manual expansion of the Changelog.

@justinmayer : I'm not sure what you want to do for reviewing my PR's... Do you want to review and merge them yourself, or are you ok with me taking the lead on these?

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,7 +4,13 @@ Jinja Filters Changelog
 2.0.0 - 2020-08-21
 ------------------
 
-Initial release as namespace plugin
+- **feature** Initial release as namespace plugin. With Pelican 4.5, this
```
Comment (justinmayer) [inline CHANGELOG.rst]: this 👉 ~~this~~

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,7 +4,13 @@ Jinja Filters Changelog
...
+  namespace plugins no longer need to be explicitly declared to be available to
+  Pelican.
+- **support** minimum versoin of Pelican is now 4.5
```
Comment (justinmayer) [inline CHANGELOG.rst]: versoin 👉 version

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,7 +4,13 @@ Jinja Filters Changelog
...
+- **support** Move plugin to the `pelican-plugins` organization on GitHub. The
```
Comment (justinmayer) [inline CHANGELOG.rst]:
```rst
`pelican-plugins` organization on GitHub 👉 `Pelican Plugins`_ organization on GitHub
```
`+`
```rst
.. _Pelican Plugins: https://github.com/pelican-plugins
```

**Review by justinmayer — COMMENTED** [review]
I pointed out a misspelling and made a one other minor suggestion.

**Comment by justinmayer** [comment]
@MinchinWeb: Thanks for the enhancements. I made a minor correction and a suggestion. ✨ 

> Do you want to review and merge them yourself, or are you ok with me taking the lead on these?

I think having the extra pair of eyes can only benefit the project, so if you don't mind, perhaps we can merge PRs after review & consensus that it's ready?

**Comment by minchinweb** [comment]
@justinmayer Considering what my spelling was like last night, another pair of eyes is a good plan.

Changes as suggested have been committed.

One other change was I bumped the *flake8* pre-commit test to run in Python 3.8 (from 3.7). I have Python 3.8 and 3.9 installed on my machine (but no longer 3.7), and otherwise, I couldn't commit anything (I guess I could have re-installed 3.7, but that seemed a little extreme, as I'd also have to re-install 3.9 to get `python` to point back to 3.9).

I'm not sure what the best plan is here...do we really want to require everyone who contributes to have multiple versions of Python installed? (pre-commit was looking for at least 3.7 and 3.8 based on the registry keys it couldn't find...) Or can be drop the requirement for a certain version of Python from the *flake8* test?

**Comment by justinmayer** [comment]
Many thanks for the updates here, which I manually merged. Regarding the `language_version` Flake8 directive in `.pre-commit-config.yaml`, I think it's _raison d'être_ has expired, so I removed it entirely in the most recent commit to the project. 💫

**Comment by minchinweb** [comment]
> I think it's raison d'être has expired, so I removed it entirely in the most recent commit to the project.

Awesome!

**Pull request closed without merging**